openproblems/tools/normalize.py

```diff
@@ -44,8 +44,9 @@ def log_scran_pooling(adata: ad.AnnData) -> ad.AnnData:
 def _cpm(adata: ad.AnnData):
     import scanpy as sc
 
-    adata.layers["counts"] = adata.X.copy()
```

Comment:
Are you removing this to make sure counts layers are always only defined in the data loaders?

Reply:
The copy here was done because the scanpy function below operates in-place. The next line makes it more explicit that this is what we are doing.